eurocodedesign/stepper/__init__.py

```diff
@@ -0,0 +1,71 @@
+"""
+Thread-safe module for collecting printable information on calculation steps
+
+Examples
+    from eurocodedesign import stepper
+    >>> with stepper.create() as st1:
+    >>>    section = "IPE300"
+    >>>    st1.step(f"Steel section {section} has height 3.0 m.")
+    >>>    # do stuff
+    >>>    st1.step(f"{section} is of cross-section type {QK}.")
+    Steel section IPE300 has height 3.0 m. IPE300 is of cross-section type QK3.
+    >>> st2 = stepper.create()
+    >>> st2.step("Add step")
+    >>> del st2
+    Add step
+"""
+from collections import deque
+from typing import Self
+
```

Comment:
Add the code below to avoid the failing python 3.10 builds just like you did for the `Units` module
`if sys.version_info >= (3, 11):
    from typing import Self
else:
    from typing_extensions import Self`

Reply:
Thank you